pyLSV2/client.py

```diff
@@ -2213,3 +2216,144 @@ def tst_record_data(
+                    signal.channel_name,
+                    signal.signal_name,
+                    signal.min_interval,
+                )
+                raise Exception("the selected interval is to small")
+
+            payload.extend(signal.to_ba())
+
+        result = self._send_recive(lc.CMD.R_OP, payload, lc.RSP.S_OP)
+        if isinstance(result, (bytearray,)) and len(result) > 0:
+            self.tst_decode_signal_details(signal_list, result)
+        else:
+            if self.last_error.e_code == 85:
+                self._logger.warning("too many signals selected: %d", len(signal_list))
+                raise Exception("too many signals selected???")
+            if self.last_error.e_code == lc.LSV2StatusCode.T_ER_OSZI_CHSEL:
+                self._logger.warning("Error setting up the channels")
+                raise Exception("Error setting up the channels")
+            raise Exception()
+        
+        # for i in signal_list:
+        #     print(i)
+
+        # step 3
+        result = self._send_recive(lc.CMD.R_DT, None, lc.RSP.S_DT)
+        if isinstance(result, (bytearray,)) and len(result) > 0:
+            ts = self.decode_timestamp(result)
+            self._logger.debug("Time on Control is %s", ts.isoformat())
+        else:
+            raise Exception()
+
+        # step 4
+
+        """
+        capture 1: one channel tree axes 3k us interval
+        "\x00\x06\xff\xff\x00\x00\x00\x00\x00\x00\x00\x00 \x00\x00\x0b\xb8"
+
+
+        capture 2: two channels 1 axes 3k us interval
+        "\x00\x06\xff\xff\x00\x00\x00\x00\x00\x00\x00\x00 \x00\x00\x0b\xb8"
+
+
+        capture 3: two channels 1 axes 3k us interval 16 k buffer
+        "\x00\x06\xff\xff\x00\x00\x00\x00\x00\x00\x00\x00 \x00\x00\x0b\xb8"
+       
+        capture 4: two channels 1 axes 6k us interval
+        "\x00\x06\xff\xff\x00\x00\x00\x00\x00\x00\x00\x00 \x00\x00\x17\x70"
+
+
+        # bytes 12 - 15 are a repeat of the interval value? 3000 and 6000
+        """
+        payload = bytearray()
+        payload.append(0x00)
+        payload.append(0x06)
+        payload.append(0xFF)
+        payload.append(0xFF)
+        payload.append(0x00)
+        payload.append(0x00)
+        payload.append(0x00)
+        payload.append(0x00)
+        payload.append(0x00)
+        payload.append(0x00)
+        payload.append(0x00)
+        payload.append(0x00)
+
+        # interval again?
+        payload.extend(struct.pack("!L", intervall_us))
+        # payload.append(0x00)
+        # payload.append(0x00)
+        # payload.append(0x0b) # -> 0B
+        # payload.append(0xb8) # -> B8
+
+        recorded_data = list()
+        content = self._send_recive(lc.CMD.R_OD, payload, lc.RSP.S_OD)
+
+        if isinstance(content, (bytearray,)) and len(content) > 0:
+            recorded_data.append(self.tst_decode_scope_reading(signal_list, content))
+
+            end = time.time() ; timer = end - start ; TCP_package_num = 0
```

Comment:
split this into three separate lines, TCP_package_num does not conform to snake_case

Reply:
Done.
TCP_package_num  is not used anymore.